backend/pkg/dbstore/dbstore.go

```diff
@@ -170,7 +170,7 @@ func (datasource *SQLiteDatasource) Init() {
 		panic(err)
 	}
 
-	stmt, err := db.Prepare("CREATE TABLE IF NOT EXISTS Schedule (id TEXT PRIMARY KEY, interval INTEGER, nextReportTime INTEGER, name TEXT, description TEXT, lookback INTEGER, reportGroupID TEXT, FOREIGN KEY(reportGroupID) REFERENCES ReportGroup(id))")
+	stmt, err := db.Prepare("CREATE TABLE IF NOT EXISTS Schedule (id TEXT PRIMARY KEY, interval INTEGER, nextReportTime INTEGER, name TEXT, description TEXT, lookback INTEGER, reportGroupID TEXT, time INTEGER, day INTEGER, FOREIGN KEY(reportGroupID) REFERENCES ReportGroup(id))")
```

Comment:
apologies - the quick and dirty way was to store exactly as returned by the front end component. So the format is something like `08:15`

```suggestion
	stmt, err := db.Prepare("CREATE TABLE IF NOT EXISTS Schedule (id TEXT PRIMARY KEY, interval INTEGER, nextReportTime INTEGER, name TEXT, description TEXT, lookback INTEGER, reportGroupID TEXT, time TEXT, day INTEGER, FOREIGN KEY(reportGroupID) REFERENCES ReportGroup(id))")
```

Reply:
totes forgot about this 😢 568c370